### ISSUE TITLE
Bug in order in which template is chosen.

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#822]
As reported by @marktastic:

Hello, I was having an issue with template file resolution in Bolt which I think may be a bug.
I was having an issue with record_template in contenttypes.yml not actually getting used when they were set.
It was instead using a twig template that was named the same as the contenttype.
For example, if my contenttype was named 'press' and I specifed the record_template to be news.twig, Bolt would choose to use press.twig over news.twig.
My expected behavior is that it should prefer to use the one I am manually specifying.
In app/src/Bolt/Content.php, I fixed this by moving lines 816-822 after the next conditional block as follows:

from this

``` php
if (isset($this->contenttype['record_template'])) {
    $templatefile = $this->app['paths']['themepath'] . "/" . $this->contenttype['record_template'];
    if (file_exists($templatefile)) {
        $template = $this->contenttype['record_template'];
        $chosen = 'contenttype';
    }
}

$templatefile = $this->app['paths']['themepath'] . "/" . $this->contenttype['singular_slug'] . ".twig";
if (is_readable($templatefile)) {
    $template = $this->contenttype['singular_slug'] . ".twig";
    $chosen = 'singular_slug';
}
```

to this

``` php
$templatefile = $this->app['paths']['themepath'] . "/" . $this->contenttype['singular_slug'] . ".twig";
if (is_readable($templatefile)) {
    $template = $this->contenttype['singular_slug'] . ".twig";
    $chosen = 'singular_slug';
}

if (isset($this->contenttype['record_template'])) {
    $templatefile = $this->app['paths']['themepath'] . "/" . $this->contenttype['record_template'];
    if (file_exists($templatefile)) {
        $template = $this->contenttype['record_template'];
        $chosen = 'contenttype';
    }
}
```

tl;dr: This makes the preference for the manually included record_template.
